frontend/src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -88,56 +118,101 @@ export class DebugInformation extends React.PureComponent<DebugInformationProps,
     }
   }
 
-  renderHealthConfig = (config: Array<any> | Object | RegExp | string) => {
-    if (Array.isArray(config)) {
-      let arr = [];
-      for (let v of config) {
-        arr.push(this.renderHealthConfig(v) as never);
-      }
-      return arr;
+  parseConfig = (key: string, value: any) => {
+    // We have to patch some runtime properties  we don't want to serialize
+    if (['cyRef', 'summaryTarget', 'token', 'username'].includes(key)) {
```

Comment:
This array of ignored keys may be nice as another constant.

Reply:
Agree, Updated